src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Automation/Peers/MenuItemAutomationPeer.cs

```diff
@@ -195,12 +198,58 @@ protected override List<AutomationPeer> GetChildrenCore()
                                 children.Add(peer);
                         }
                     }
+
+                    return children;
                 }
             }
 
+            return GetChildrenFromVisualTree();
```

Comment:
This changes the current behavior, correct? Before the PR, the children contained the visual tree _and_ children. After the PR, it contains visual tree _or_ children.

Reply:
I tried to create the same behavior like all other controls that support grouping. I have to check how hte behavior first. I'm a bit out of topic since my last changes.